src/test/unit/sync_http/test_sync_hook.py

```diff
@@ -111,6 +111,21 @@ def lambda_test_function():
     assert http_spans[0]["info"]["httpInfo"]["request"]["uri"] == "www.google.com/?q=123"
 
 
+def test_uri_requests():
+    @lumigo_tracer(token="123")
+    def lambda_test_function():
+        conn = http.client.HTTPConnection("www.google.com")
```

Comment:
Don't we use a package to mock the API call?

Reply:
nope.
like in the rest of the tests in this file, we're actually do API requests (just like our customers) and we check that the tracer parse them well :)